packages/client/components/ActivityLibrary/IsOneOnOneTeamExists.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {Suspense} from 'react'
+import graphql from 'babel-plugin-relay/macro'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
+import {
+  IsOneOnOneTeamExistsQuery,
+  CreateOneOnOneTeamInput
+} from '../../__generated__/IsOneOnOneTeamExistsQuery.graphql'
+import useQueryLoaderNow from '../../hooks/useQueryLoaderNow'
+import {renderLoader} from '../../utils/relay/renderLoader'
+import {LoaderSize} from '../../types/constEnums'
+
+const isOneOnOneExistsQuery = graphql`
+  query IsOneOnOneTeamExistsQuery($oneOnOneTeamInput: CreateOneOnOneTeamInput!) {
+    isOneOnOneTeamExists(oneOnOneTeamInput: $oneOnOneTeamInput) {
+      team {
+        id
+        name
+      }
+    }
+  }
+`
+
+interface ComponentProps {
+  queryRef: PreloadedQuery<IsOneOnOneTeamExistsQuery>
+  name: string
+}
+
+const IsOneOnOneTeamExistsComponent = (props: ComponentProps) => {
```

Comment:
+1 we usually create a separate file for a component like this. I think that'd be more consistent with the rest of the codebase